handlers/handlers.go

```diff
@@ -121,3 +221,79 @@ func getRequestBody(request *http.Request) []byte {
 
 	return body
 }
+
+
+// Token Util
+const oneWeekInMinutes = 60 * 24 * 7
+const cerealNotesCookieName = "CerealNotesToken"
+
+func parseTokenFromString(tokenString string) (*jwt.Token, error) {
+	// Parse the token
+	token, err := jwt.ParseWithClaims(
+		strings.TrimSpace(tokenString),
+		&CerealNotesClaims{},
+		func(token *jwt.Token) (interface{}, error) {
+			return tokenSigningKey, nil
+		})
+	return token, err
+}
+
+func createTokenAsString(
+	userId models.UserId,
+	expirationTimeInMinutes int64,
+) (string, error) {
+	// Create the Claims
+	claims := CerealNotesClaims{
+		userId,
+		jwt.StandardClaims{
+			ExpiresAt: time.Now().Unix() + (expirationTimeInMinutes * 60),
```

Comment:
wait so the cookie has the expiration time as a timestamp, and in the token it's seconds since the epoch? I know we don't get to decide those things, but jesus

Reply:
unfortunately you are correct.